app/lib/notifications.server.ts

```diff
@@ -0,0 +1,37 @@
+import { createTransport, type SendMailOptions } from 'nodemailer';
+import { secrets } from 'docker-secret';
+import logger from './logger.server';
+
+const { NODEMAILER_USER } = process.env;
+const { NODEMAILER_CLIENT_ID, NODEMAILER_ACCESS_TOKEN } = secrets ?? {};
+
+const initializeTransport = () => {
+  if (!NODEMAILER_USER || !NODEMAILER_CLIENT_ID || !NODEMAILER_ACCESS_TOKEN) {
+    throw new Error(
+      'Missing Nodemailer environment variables or docker secrets. Skipping nodemailer configuration'
+    );
+  }
+  const transporter = createTransport({
+    service: 'Outlook365',
+    auth: {
+      type: 'OAuth2',
+      user: NODEMAILER_USER, // email address for notifications
+      clientId: NODEMAILER_CLIENT_ID,
+      accessToken: NODEMAILER_ACCESS_TOKEN,
+      expires: 1484314697598,
```

Comment:
Should this be lowered to something saner?

Reply:
Would 3 or 5 days be more appropriate?

Reply:
Scaled down to 60hrs, for now.